### ISSUE TITLE
Pretibial hyperpgimentation

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#9560]
**Preferred term label:**
Pretibial hyperpgimentation

**Synonyms**

**Definition (free text, please give PubMed ID)**
Darkening of the area of skin that overlies the shin.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
130080 
PMID: 27745832
24/25

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Also gum fragility 20/21  130080
PMID: 27745832

**Comment by pnrobinson** [comment]
added
